parm/FV3LAM_wflow.xml

```diff
@@ -141,19 +189,39 @@ Directories and files.
 <!ENTITY LOAD_MODULES_RUN_TASK_FP "{{ load_modules_run_task_fp }}">
 <!ENTITY EXPTDIR                  "{{ exptdir }}">
 
+<!ENTITY OBSPATH                  "{{ obspath }}">
+<!ENTITY NWGES_BASEDIR            "{{ nwges_basedir }}">
+<!ENTITY ENSCTRL_COMIN_BASEDIR    "{{ ensctrl_comin_basedir }}">
+<!ENTITY ENSCTRL_NWGES_BASEDIR    "{{ ensctrl_nwges_basedir }}">
+<!ENTITY ENSCTRL_COMOUT_BASEDIR   "{{ ensctrl_comout_basedir }}">
+<!ENTITY ENSCTRL_COMOUT_DIR       "{{ ensctrl_comout_basedir }}/{{ run }}.<cyclestr>@Y@m@d@H</cyclestr>">
+<!ENTITY RRFSE_FG_ROOT            "{{ rrfse_nwges_basedir }}">
+{%- if is_rtma %}
+<!ENTITY FG_ROOT                  "{{ fg_rootdir }}">
+{%- else %}
+<!ENTITY FG_ROOT                  "{{ nwges_basedir }}">
+{%- endif %}
+
+<!-- Temporarily turn off fcst_fv3lam subdirectory usage -->
+<!-- until run_fcst of rrfs_dev is merged                -->
+{%- if do_rrfs_dev %}
+<!ENTITY FCST_DIR                  "fcst_fv3lam/">
+{%- else %}
+<!ENTITY FCST_DIR                  "">
+{%- endif %}
```

Comment:
This will be removed later once everything is merged. If we start using `fcst_fv3lam` and `fcst_fv3lam_spinup` directories for the forecast output right from the beginning it would require modifying forecast/post etc steps from the very beginning making reviews complex. Only minimal changes are made in those scripts to account for the name change from `run_fcst` to `run_fcst_prod` in this PR.